QualificationsFramework/Examples/ACQF.json

```diff
@@ -0,0 +1,184 @@
+			"skos:note": { "en": "In the context of ACQF \"Skills\" refer to the ability to use knowledge to respond to information and address problems. Skills include cognitive, communication, digital, green, innovation, practical and social skills." }
+		},
+		{
+			"@id": "res:facetAutonomy",
+			"@type": "skos:Concept",
+			"skos:prefLabel": { "en": "Autonomy and Responsibility" },
+			"skos:definition": { "en": "Refers to the context and extent of the application of autonomy and responsibility. The learning outcomes at all ten ACQF levels relate to formal, non-formal and informal learning. The autonomy and responsibility domain comprises 3 sub-domains:\n- the context in which autonomy and responsibility is applied\n- the extent to which autonomy is applied\n- the extent to which responsibility is applied.\nResponsibility as a domain of level descriptors has three elements: self, group outcomes, and resources." },
+			"skos:note": { "en": "In the context of ACQF \"Autonomy and responsibility\" refers to the context and extent of the application of autonomy and responsibility" }
+		},
+		{
+			"@id": "res:acqfLevels",
+			"@type": "asn:ProgressionModel",
+			"ceasn:name": { "en": "ACQF Level Descriptors" },
+			"ceasn:definition": { "en": "Each of the ten ACQF levels represents learning outcomes related to formal, non-formal and informal learning at that level. The ACQF level descriptors capture how the three domains of knowledge, skills and autonomy and responsibility (A&R) increase in breadth, depth and complexity when moving from lower to higher levels and are supported by a glossary explaining the terms." },
+			"ceasn:hasGlossary": [
+				"res:acqfGlossary"
+			],
+			"asn:hasProgressionLevel": [
+				"res:acqfLevel01",
+				"res:acqfLevel02",
+				"etc"
+			]
+		},
+		{
+			"@id": "res:acqfLevel01",
+			"@type": "asn:ProgressionLevel",
+			"skos:prefLabel": { "en": "Level 1" },
+			"skos:definition": { "en": "The learning outcomes related to formal, non-formal, and informal learning at this level include:" }
+			"ceasn:facetedDescription": [
+				"res:knowledge_01",
+				"res:skill_01",
+				"res:autonomy_01"
+			]
+		},
+		{
+			"@id": "res:knowledge_01",
+			"@type": "ceasn:Competency",
+			"ceasn:competencyText": { "en": "Simple knowledge, literacy, and numeracy" },
+			"???:facetType": [
+				"res:facetKnowledge"
+			],
+			"ceasn:isPartOf": [
+				"res:acqf"
+			]
+		},
+		{
+			"@id": "res:skill_01",
+			"@type": "ceasn:Competency",
+			"ceasn:competencyText": { "en": "Simple communication, cognitive and practical skills required to follow simple instructions, and use simple, repetitive solutions to address simple problems" },
```

Comment:
The word `Simple` repeats four times. Necessary? :)

Reply:
Not ours to fix.